### ISSUE TITLE
Full support for special characters

### ISSUE THREAD

**Issue by gruhn** [bryphe/oni#711]
Oni has some weird behavior, when dealing with special characters. Typing `öäüß` in INSERT mode works but typing their upper-case counterparts `ÖÄÜ` has no effect.

If files, contain these characters, they can be displayed. Looks like the upper-case `Ö` got converted to a reqular `O` though. Not sure, I guess I need glasses...

![bildschirmfoto vom 2017-09-16 10-37-25](https://user-images.githubusercontent.com/26570572/30510803-623a9088-9acb-11e7-8447-ea0e6badaeec.png)

File content:

```
ö : Ö
ä : Ä
ü : Ü
ß
```

In fuzzy finder I'm able to type `ÖÄÜ` but encoding breaks for files with those characters:  

![bildschirmfoto vom 2017-09-16 10-57-29](https://user-images.githubusercontent.com/26570572/30510906-f3911d8e-9acd-11e7-9504-865898581a79.png)

The file is actually called `Überblick.md`

**Comment by extr0py** [comment]
Thanks for the issue, @gruhn ! The special character issue seems like it is the same as #644. The fuzzy finder issue hasn't been tracked yet, it looks like there is a problem with encoding the process output. 

A couple of question that will help once I start looking at this:
- Which OS is this?
- Is the `Überblick.md` file in a git repo? This can impact the behavior of the fuzzy-find.

**Comment by gruhn** [comment]
You're right. This file is in a git repo. I quickly checked in a regular folder and fuzzy-finding the file was not a problem. This is what I experienced on my laptop running  __Linux Mint 18.1__ with  __Oni 0.2.9__ installed. 

But now that you mentioned it, I also tested Oni on another machine of mine running __Linux Mint 17.2__ and __Oni 0.2.8__. Now checkout this:

![out-1-1-1](https://user-images.githubusercontent.com/26570572/30826014-2ef33826-a235-11e7-9eba-ca79c0c022ff.gif)

Lower-case characters aren't an issue here either. `ÜÖ` is replaced with `UO` but only in the first line. In the second line and below, all of them work but the dots are still visible, even when the character and the hole line is deleted. They disappear when I delete the last character of the first line though.

I then upgraded to 0.2.9 on this computer and as far as I can tell, it's the same behavior now as on my laptop, so I can't type `ÜÄÖ` at all anymore.
